### PR TITLE
Saving new student record

### PR DESCRIPTION
**Pull request by winterj**
Also fixing pasted in family code.

### CODE REVIEW

File: app/Http/Controllers/StudentsController.php [hunk 1]
```diff
@@ -42,7 +43,27 @@ public function create(Family $family)
...
+            'Sex' => 'required',
+            'family_id' => 'required',
+            'Birthday' => 'required',
+            'MedicalConditions' => 'required',
+            'PaperWaiver' => 'required',
+            'OnlineWaiverAccepted' => 'required',
+            'Performing' => 'required',
+            'ThirdPartyID' => 'required'
+        ]);
+        $validated['Birthday'] = Carbon::parse($validated['Birthday']);
```
Comment (Hobstweedl) [inline app/Http/Controllers/StudentsController.php:58]: This whole validated object is news to me.

File: resources/views/students/create.blade.php [hunk 2]
```diff
@@ -75,7 +74,7 @@
             <div class="field-body">
                 <div class="field">
                     <div class="control is-expanded">
-                        <input class="input" type="text" id="BirthDate" name="BirthDate" placeholder="Birth Date" value="{{\Carbon\Carbon::parse($student->Birthday)->format('m/d/Y')}}" required/>
+                        <input class="input" type="text" id="Birthday" name="Birthday" placeholder="Date of Birth" value="{{\Carbon\Carbon::parse($student->Birthday)->format('m/d/Y')}}" required/>
```
Comment (Hobstweedl) [inline resources/views/students/create.blade.php:77]: If possible, try to process the data in the controller and pass a formatted value to the view.

**Review by Hobstweedl — APPROVED** [review]
Looks good. Are you going to provide error labels for incorrect fields?

**Comment by winterj** [comment]
What do you mean? Like for validation?

**Comment by Hobstweedl** [comment]
Yes for the validation. If the user doesn't know what's wrong with the form they can't fix it and submit it

**Comment by winterj** [comment]
Right now I just have the flash message. But yes, the forms still need work. Right now the validations are all just required.

**Pull request merged**